core/src/main/java/org/bitcoinj/core/BitcoinSerializer.java

```diff
@@ -96,6 +97,7 @@
         names.put(ChainLockSignature.class, "clsig");
         names.put(SendHeadersMessage.class, "sendheaders");
         names.put(GetMasternodePaymentRequestSyncMessage.class, "mnget");
+        names.put(CreditFundingTransaction.class, "tx");
```

Comment:
Added to allow a `CreditFundingTransaction` to be sent using `Peer.sendMessage`

The names map says what message ("tx" in this case is associated with what class).